sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -58,7 +58,7 @@ your resource and a `credential` that allows you access:
 ```python
 from azure.ai.textanalytics import TextAnalyticsClient
 
-text_analytics = TextAnalyticsClient(endpoint="https://westus2.api.cognitive.microsoft.com/", credential=credential)
```

Comment:
Do we need to mention that this service isn't supported in the west us location, yet?

Reply:
It is supported in westus2. I link out to regional support somewhere in the README, and I'd rather the service keep that list up-to-date.